packages/falcon-ui-kit/src/Locale/LocalePicker.tsx

```diff
@@ -29,8 +29,9 @@ export const LocalePickerInnerDOM: React.SFC<LocalePickerProps> = ({ items, valu
   </Dropdown>
 );
 LocalePickerInnerDOM.propTypes = {
+  // @ts-ignore https://github.com/facebook/prop-types/issues/296
   items: PropTypes.arrayOf(
-    PropTypes.exact({
+    PropTypes.shape({
```

Comment:
so this should be reverted i think 
```suggestion
    PropTypes.exact({
```